packages/ui/src/components/SnackBar/Snackbar.tsx

```diff
@@ -0,0 +1,41 @@
+import { Stack, Snackbar as MuiSnackbar, styled } from "@mui/material";
+import { useSnackStack } from "./SnackStackProvider";
+import ToastBar from "../ToastBar";
+
+interface Props {
+  className?: string
+}
+
+const Snackbar: React.FC  = ({ className }: Props) => {
+  const { toastsPack } = useSnackStack();
+  const firstToast = toastsPack[0];
+
+  return (
+    <MuiSnackbar
+      className={className}
+      open={!!firstToast}
+      autoHideDuration={12000}
```

Comment:
This doesn't seem to have any effect. Same for the `transitionDuration`. This doesn't sounds needed